src/OrchardCore/OrchardCore.Navigation.Core/NavigationHelper.cs

```diff
@@ -11,6 +10,9 @@ namespace OrchardCore.Navigation
 {
     public class NavigationHelper
     {
+        public static bool IsAdminMenu(string name)
```

Comment:
Add a note about this to the release notes, because it'll also be the recommended way for module authors.

Reply:
yes. I'll add that to the release notes.